### ISSUE TITLE
Problem with build 

### ISSUE THREAD

**Issue by kordi83** [universam1/iSpindel#365]
Hi,

I have build 2 different ispindels with 4.0 board  Joey Joe board. When i flash it with 6.2 i can`t get it in config mode. When i use 5.8 it can`t safe any info. in the info shows that my temp sensor is not found and it is at -127. I have reordered and used sensor form another supplier.  Please help

2.2.1(cfd48f3)
Worker run!
mounting FS... ERROR: failed to mount FS!

ERROR config corrupted
scanning for OW device on pin: 5
No devices found!
scanning for OW device on pin: 12
No devices found!
scanning for OW device on pin: 116
No devices found!
scanning for OW device on pin: 121
No devices found!
ERROR: cannot find a OneWire Temperature Sensor!
Boot-Mode: Deep-Sleep Wake
...........
           ERROR no Wifi credentials

going to Config Mode

**Comment by mnandres** [comment]
Hi, i have a similiar problem. some help too find the problem?

**Comment by thegreatgunbantoad** [comment]
It's highly unlike that the temperature sensor is not working.  I've never had one fail, bad gyros and poorly soldered Wemos boards I have.  Check the joints on the Wemos' daughter board.  Also check that the temp sensor is fitted correctly to the correct side of the board.  It does look though like you have some other underlying issue.

**Comment by LloydPickering** [comment]
I am having exactly the same problem, except I have plugged everything except battery charger into a breadboard to diagnose (except the GY521 which is already soldered on). I have continuity checked the connections where possible and it all seems fine. I have tried reversing the temp sensor.

Picture of breadboard attached. I've tried to use the same colour scheme as the circuit diagram for clarity. ![IMG_20200506_190624](https://user-images.githubusercontent.com/248082/81212930-8fb08780-8fcd-11ea-9993-07c6e1c015a8.jpg)

When flashing 6.2 I get the same messages as the original poster.

When flashing 5.8 I can go in and edit the config. I can see the angle of inclination seems to be working, temp sensor is not, showing -127 (I guess it defaults to min value on a signed char). When I save settings, the output complains the config is corrupt and resets to being an AP. Here is the log from that (replacing the wifi ssids with [redacted]):

FW 5.8.6
2.2.1(cfd48f3)
Worker run!
Boot-Mode: 5
mounting FS... ERROR: failed to mount FS!

ERROR config corrupted
...........
           ERROR no Wifi crenals

goingto Config Mode
ERROR: OW DISCONNECTED
ERROR: OW DISCONNECTED
*WM: Scan done
*WM: DUP AP: [redacted]
*WM: DUP AP: [redacted]
*WM: DUP AP: [redacted]
*WM: DUP AP: [redacted]
*WM: DUP AP: [redacted]
*WM: DUP AP: [redacted]
*WM: Adding parameter
*WM: name
*WM: Adding parameter
*WM: sleep
*WM: Adding parameter
*WM: vfact
*WM: Adding parameter
*WM:
*WM: Adding parameter
*WM:
*WM: Adding parameter
*WM: tempscale
*WM: Adding aameter
*WM:
*WM: Adding parameter
*WM:
*WM: Adding parameter
*WM: selAPI
*WM: Adding parameter
*WM: token
*WM: Adding parameter
*WM: server
*WM: Adding parameter
*WM: port
*WM: Adding parameter
*WM: url
*WM: Adding parameter
*WM: db
*WM: Adding parameter
*WM: job
*WM: Adding parameter
*WM: instance
*WM: Adding parameter
*WM:
*WM: Adding parameter
*WM: POLYN
started Portal
*WM: SET AP
*WM:
*WM: Configuring accs ont..
*WM: iSpindel
*WM: AP IP address:
*WM: 192.168.4.1
*WM: HTTP server started
ERROR: OW DISCONNECTED
*WM: Request redirected to pve▒▒х▒5
                                   ERROR: OW DISCONNECTED
ERROR: OW DISCONNECTED
*WM: Request redirected t▒▒ѥٕ portal
ERROR: OW DISCONNECTED
